doc/Overview/Mvux/Advanced/Pagination.md

```diff
@@ -0,0 +1,395 @@
+---
+uid: Overview.Mvux.Advanced.Pagination
+---
+
+# Pagination
+
+There are several ways to paginate data.
+
+> [!NOTE]  
+> The source-code for the sample app demonstrated in this section can be found [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/PaginationPeopleApp).
+
+## Incremental loading
+
+The easiest and most straight-forward is use the built-in incremental loading functionality that some controls (e.g. `ListView`, `GridView`) offer via the [`ISupportIncrementalLoading`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.data.isupportincrementalloading) interface the paginated Feed implements.
+
+For the Pagination example we'll also use the *PeopleApp* example used above in [Selection](xref:Overview.Mvux.Advanced.Selection), but you can find the full code [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/PaginationPeopleApp).
+
+### Service
+
+Let's change the `GetPeopleAsync` method in the *PeopleService.cs* file to support pagination:
+
+```csharp
+namespace PaginationPeopleApp;
+
+public partial record Person(string FirstName, string LastName);
+
+public interface IPeopleService
+{
+    ValueTask<IImmutableList<Person>> GetPeopleAsync(uint pageSize, uint pageIndex, CancellationToken ct);
```

Comment:
Do these need to be uint? If not, please just use int as this is what most devs will default to

Reply:
Added comment explaining why `uint` is used.